data/ledger.json

```diff
@@ -7718,9 +7718,9 @@
 {"action":{"alias":{"address":"N\u0000sourcecred\u0000discord\u0000MEMBER\u0000user\u0000969380967234031668\u0000","description":"discord/Sacee#3206"},"identityId":"V1p0nQz0tmhMaUw4SwFy2A","type":"ADD_ALIAS"},"ledgerTimestamp":1652508276190,"uuid":"osqtR1jIeAPhaksrLlJbKQ","version":"1"}
 {"action":{"identity":{"address":"N\u0000sourcecred\u0000core\u0000IDENTITY\u00002vuEjzfkRAuJPraT9mtxsg\u0000","aliases":[],"id":"2vuEjzfkRAuJPraT9mtxsg","name":"Cryptojacky","subtype":"USER"},"type":"CREATE_IDENTITY"},"ledgerTimestamp":1652508276191,"uuid":"8wes3VtlQdt4kmcg1u9W1A","version":"1"}
 {"action":{"alias":{"address":"N\u0000sourcecred\u0000discord\u0000MEMBER\u0000user\u0000971784972820185118\u0000","description":"discord/Cryptojacky#9697"},"identityId":"2vuEjzfkRAuJPraT9mtxsg","type":"ADD_ALIAS"},"ledgerTimestamp":1652508276191,"uuid":"lmiQB1xnDhvUaam9x2ruXQ","version":"1"}
-{"action":{"currency":{"chainId":"1","tokenAddress":"0x333A4823466879eeF910A04D473505da62142069","type":"EVM"},"id":"BtlDOgLvfl76rb1jGwi1oQ","payoutAddress":"0x4e3072f7b5c075ea5fdeb423da95312c4b99dc22","type":"SET_PAYOUT_ADDRESS"},"ledgerTimestamp":1652508453019,"uuid":"2FArHEDBcgKBp44XNOpotQ","version":"1"}
-{"action":{"currency":{"chainId":"1","tokenAddress":"0x333A4823466879eeF910A04D473505da62142069","type":"EVM"},"id":"LMUEWiZux2Q8iAcKKy8gGg","payoutAddress":"0x4e3072f7b5c075ea5fdeb423da95312c4b99dc22","type":"SET_PAYOUT_ADDRESS"},"ledgerTimestamp":1652508458077,"uuid":"Q1AmEyAVEg4P5cxyb8ejIA","version":"1"}
-{"action":{"currency":{"chainId":"1","tokenAddress":"0x333A4823466879eeF910A04D473505da62142069","type":"EVM"},"id":"o4RCyNUm0AM3DF6NqrRLnQ","payoutAddress":"0x4e3072f7b5c075ea5fdeb423da95312c4b99dc22","type":"SET_PAYOUT_ADDRESS"},"ledgerTimestamp":1652508470072,"uuid":"8uRBXGtmSlM0WynJgnCkQw","version":"1"}
+{"action":{"currency":{"chainId":"1","tokenAddress":"0x333A4823466879eeF910A04D473505da62142069","type":"EVM"},"id":"BtlDOgLvfl76rb1jGwi1oQ","payoutAddress":"0x4e3072f7b5C075EA5FdEb423DA95312C4B99dc22","type":"SET_PAYOUT_ADDRESS"},"ledgerTimestamp":1652508453019,"uuid":"2FArHEDBcgKBp44XNOpotQ","version":"1"}
+{"action":{"currency":{"chainId":"1","tokenAddress":"0x333A4823466879eeF910A04D473505da62142069","type":"EVM"},"id":"LMUEWiZux2Q8iAcKKy8gGg","payoutAddress":"0x4e3072f7b5C075EA5FdEb423DA95312C4B99dc22","type":"SET_PAYOUT_ADDRESS"},"ledgerTimestamp":1652508458077,"uuid":"Q1AmEyAVEg4P5cxyb8ejIA","version":"1"}
+{"action":{"currency":{"chainId":"1","tokenAddress":"0x333A4823466879eeF910A04D473505da62142069","type":"EVM"},"id":"o4RCyNUm0AM3DF6NqrRLnQ","payoutAddress":"0x4e3072f7b5C075EA5FdEb423DA95312C4B99dc22","type":"SET_PAYOUT_ADDRESS"},"ledgerTimestamp":1652508470072,"uuid":"8uRBXGtmSlM0WynJgnCkQw","version":"1"}
```

Comment:
@johnmark13 The change from lower-case to checksummed is probably because you manually ran `sourcecred load` before running `yarn update-ledger-from-chain`?  It makes it difficult to review the changes, so can we make a PR without these upper-case/lower-case diffs?

Reply:
I don't think so, we definitely cannot easilly split to multiple PRs. I wonder if SourceCred automatically converts the case to checksummed when setting the payout address, and we have them in lower case because of that script you ran before? If we let the ledger hold checksummed addresses it won't update them after the first time making PRs more simple.